omisego-sdk/src/main/java/co/omisego/omisego/extension/Density.kt

```diff
@@ -16,4 +15,4 @@ import android.content.res.Resources
  * This simplifies code dealing with dimensions.
  */
 inline val Int.dp: Float
-    get() = (this * Resources.getSystem().displayMetrics.density)
\ No newline at end of file
+    get() = (this * Resources.getSystem().displayMetrics.density)
```

Comment:
Need to be careful with this, value could be wrong when using multiple screens :)